src/index.ts

```diff
@@ -552,6 +559,90 @@ export interface SearchResponse {
   activePromotions: ActivePromotion[];
 }
 
+/**
+ * TextPosition describes the position of text in a box.
+ *
+ *  - TEXT_POSITION_UNSPECIFIED: No position specified.
+ *  - TEXT_POSITION_CENTER: The text is positioned in the horizontal and vertical center.
+ *  - TEXT_POSITION_TOP_LEFT: The text is positioned in the top left corner.
+ *  - TEXT_POSITION_TOP_RIGHT: The text is positioned in the top right corner.
+ *  - TEXT_POSITION_BOTTOM_LEFT: The text is positioned in the bottom left corner.
+ *  - TEXT_POSITION_BOTTOM_RIGHT: The text is positioned in the bottom right corner.
+ * @export
+ * @enum {string}
+ */
+export enum TextPosition {
+  Unspecified = "TEXT_POSITION_UNSPECIFIED",
+  Center = "TEXT_POSITION_CENTER",
+  TopLeft = "TEXT_POSITION_TOP_LEFT",
+  TopRight = "TEXT_POSITION_TOP_RIGHT",
+  BottomLeft = "TEXT_POSITION_BOTTOM_LEFT",
+  BottomRight = "TEXT_POSITION_BOTTOM_RIGHT",
+}
+
+export interface Banner {
+  /**
+   * The description of the banner, displayed in sub-head font.
+   * @type {string}
+   * @memberof Banner
+   */
+  description?: string;
```

Comment:
Are all of these fields actually optional?
It seems to me that the banner itself may be optional but if a banner is present, many of these fields are required. No?

Reply:
We took the interface from the generated code. However, it's true that only text fields are optional, the rest are required to create a banner. Do you think we should make an update to reflect that?

Reply:
Fair enough. Let's leave them as optional.